src/authentication.js

```diff
@@ -8,90 +12,86 @@ export const authenticationData = {
 };
 
 /**
- * Set authenticationData.firstFactors from the authentication object
- * @param {Object} authentication
- * {
- *   firstFactors,
- *   secondFactors
- * }
- * @returns
+ * Set the cookies from a tokens object, and add to the local store.
+ * @param {Object} tokens
  */
-export function setFirstFactors(authentication) {
-  // If we're not initialized, there are no first factors.
-  if (!store.tenantId) {
-    console.warn(
-      "setFirstFactors: tried to set factors without a tenantId set."
-    );
-    return;
+export function setCookiesAndTokens(tokens) {
+  setCookie(tokens.access.value, tokens.access.cookieOptions, "access");
+  setCookie(tokens.id.value, tokens.id.cookieOptions, "id");
+  if (tokens.refresh && tokens.refresh.value) {
+    setCookie(tokens.refresh.value, tokens.refresh.cookieOptions, "refresh");
```

Comment:
Heads up that the diffs on this file don't make any sense.

`setCookiesAndTokens` and `handleLoginResponse` were added, and all other methods were moved to `mfa.js`